### PR TITLE
Update DeCiFer to release v1.0.0

### PR DESCRIPTION
**Pull request by simozacca**
We would like to kindly request to update the recipe for DeCiFer for the new related release v1.0.0.

Thank you

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by fanninpm** [comment]
See #27886

**Comment by simozacca** [comment]
> See #27886

@fanninpm Thank you for this, I was unfortunately not aware that there is an automatic update system in place. Anyway, the only different of the recipe here is that I have changed `python` -> `python <3`. Since this change was not necessarily needed, we can proceed with either of the two pull requests. Could you please recommend how to proceed further?

**Comment by fanninpm** [comment]
> > See #27886
> 
> @fanninpm Thank you for this, I was unfortunately not aware that there is an automatic update system in place. Anyway, the only different of the recipe here is that I have changed `python` -> `python <3`. Since this change was not necessarily needed, we can proceed with either of the two pull requests. Could you please recommend how to proceed further?

I took a look at your repository and noticed that you were using [a version of Python that is no longer officially supported](https://www.python.org/doc/sunset-python-2/). If, _through extensive testing_, you have determined that your software can run fine under Python >= 3.6, then you could probably just close this pull request. Alternatively, you can wait until that other PR gets merged, and then rebase your PR against a newly fetched master branch (let me know if you need further guidance with doing this).

**Comment by simozacca** [comment]
> > > See #27886
> > 
> > 
> > @fanninpm Thank you for this, I was unfortunately not aware that there is an automatic update system in place. Anyway, the only different of the recipe here is that I have changed `python` -> `python <3`. Since this change was not necessarily needed, we can proceed with either of the two pull requests. Could you please recommend how to proceed further?
> 
> I took a look at your repository and noticed that you were using [a version of Python that is no longer officially supported](https://www.python.org/doc/sunset-python-2/). If, _through extensive testing_, you have determined that your software can run fine under Python >= 3.6, then you could probably just close this pull request. Alternatively, you can wait until that other PR gets merged, and then rebase your PR against a newly fetched master branch (let me know if you need further guidance with doing this).

Thanks, I will wait for the merging of the other and evaluate the need to merge this PR.

File: recipes/decifer/meta.yaml [hunk 1]
```diff
@@ -22,12 +22,12 @@ requirements:
     - cmake >=3.9
     - {{ compiler('cxx') }}
   host:
-    - python
+    - python <3
```
Comment (dpryan79) [inline recipes/decifer/meta.yaml:25]: This is already done in line 13.

**Pull request closed without merging**